### ISSUE TITLE
Cordova plugin cannot install when short name attribute used in config.xml for project

### ISSUE THREAD

**Issue by kd8ssq** [BranchMetrics/cordova-ionic-phonegap-branch-deep-linking#403]
On a new app build, if I try to add the branch-cordova-sdk plugin I receive the following error: "Error: ENOENT: no such file or directory, open 'platforms/ios/[object Object]/[object Object]-Info.plist'" (see the full output below).  

I've removed the platforms and re-added them, then when I try to add the plugin it errors out.  I've narrowed it down to the [short name entry in the config.xml](https://cordova.apache.org/docs/en/latest/config_ref/#name) file which allows you to change the text under the app icon.  If I remove the short name attribute from the config.xml file, the plugin is added properly.  

Can this attribute be taken into account?  I need to be able to shorten the text under the app icon as my app name is too long to display.

Thanks,

Adam

```
Installed platforms:
  android 6.3.0
  ios 4.5.4
```

```
NMApps:fc dev$ cordova plugin add branch-cordova-sdk
Installing "branch-cordova-sdk" for android
Installing "branch-cordova-sdk" for ios
Failed to install 'branch-cordova-sdk': Error: ENOENT: no such file or directory, open 'platforms/ios/[object Object]/[object Object]-Info.plist'
    at Error (native)
    at Object.fs.openSync (fs.js:634:18)
    at Object.fs.readFileSync (fs.js:502:33)
    at readPlist (/Apps/fc/plugins/branch-cordova-sdk/src/scripts/ios/updatePlist.js:34:15)
    at Object.addBranchSettings (/Apps/fc/plugins/branch-cordova-sdk/src/scripts/ios/updatePlist.js:16:15)
    at /Apps/fc/plugins/branch-cordova-sdk/src/scripts/hooks/beforePrepare.js:20:18
    at Array.forEach (native)
    at run (/Apps/fc/plugins/branch-cordova-sdk/src/scripts/hooks/beforePrepare.js:18:15)
    at runScriptViaModuleLoader (/usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:188:18)
    at runScript (/usr/local/lib/node_modules/cordova/node_modules/cordova-lib/src/hooks/HooksRunner.js:164:16)
Error: ENOENT: no such file or directory, open 'platforms/ios/[object Object]/[object Object]-Info.plist'

```

**Comment by ethanneff** [comment]
Hello,

Thanks for bringing this to my attention. This has been fixed in b90519e1feecb35af484be27a676ed0a56f2cf7d. I would recommend updating to the latest version `2.6.19` to get this fix. You can use this [guide](https://github.com/BranchMetrics/cordova-ionic-phonegap-branch-deep-linking#compiling-updating-the-branch-sdk) to update your Cordova Branch SDK.

Thanks,

**Comment by kd8ssq** [comment]
Works great!  Thanks for your quick work.
